src/test/java/com/maestrano/account/RecurringBillTest.java

```diff
@@ -1,37 +1,28 @@
 package com.maestrano.account;
 
-import static net.jadler.Jadler.closeJadler;
```

Comment:
Would be awesome to find the same order as eclipse
https://stackoverflow.com/questions/14716283/is-it-possible-for-intellij-to-organize-imports-the-same-way-as-in-eclipse